VLC for iOSUITests/VLCiOSTestMenu.swift

```diff
@@ -0,0 +1,81 @@
+/*****************************************************************************
+ * VLCiOSTestMenu.swift
+ * VLC for iOSUITests
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+import XCTest
+
+class VLCiOSTestMenu: XCTestCase {
+    let app = XCUIApplication()
+    var helper: TestHelper!
+
+    override func setUp() {
+        super.setUp()
+
+        XCUIDevice.shared.orientation = .portrait
+        setupSnapshot(app)
+        helper = TestHelper(lang: deviceLanguage, target: VLCiOSTestMenu.self)
+        app.launch()
```

Comment:
This here would actually be a better place for the status magic (but remember to disable it again in tearDown()).